### ISSUE TITLE
License discrepancy

### ISSUE THREAD

**Issue by sennalen** [jcrist/msgspec#677]
The github sidebar says it's under BSD 3, but the LICENSE file says "All rights reserved"

**Comment by jcrist** [comment]
That's part of the standard license text (see e.g. https://fedoraproject.org/wiki/Licensing:BSD#New_BSD_(no_advertising,_3_clause)). I am not a lawyer, but my understanding is you should read licenses as "the author retains all rights, except the rights enumerated below". From [this post here](https://opensource.stackexchange.com/questions/9427/with-bsd-3-do-i-need-to-propagate-an-all-rights-reserved-statement) it sounds like the "All rights reserved" used to be required language to imply all unstated rights were reserved by the owner, but in modern copyright law that's just implicit so the line is kind of redundant now.

In summary:
- Yes, `msgspec` is BSD 3 licensed.
- That line is part of (at least some versions) of the standard BSD 3 license text. I just copy-pasted it when I created the repo years ago.
- It seems like that line is redundant in modern times, but it's also harmless.
